app/scripts/modules/core/src/entityTag/alerts/notificationCategories.ts

```diff
@@ -0,0 +1,59 @@
+import { Dictionary, keyBy } from 'lodash';
+
+export interface INotificationCategory {
+  [key: string]: string|number;
```

Comment:
No.  I had that there for some reason at one point (access arbitrary props by key, perhaps?) but it's no longer necessary.